tests/test_actions.py

```diff
@@ -138,6 +139,45 @@ def setUp(self):
 
         self.data = pd.DataFrame.from_dict(data)
 
+        np.random.seed(42)
```

Comment:
I presume for this particular seed none of the ellipticity magnitudes exceed 1? If so, it might be good to state that in a comment, saying that there's no guard against such a thing.